IntegrationAdaptersTests/RegisterHospitalTests.cs

```diff
@@ -20,49 +20,32 @@ public void Is_pharmacy_already_existing_on_hospital()
         {          
             ApiService service = new ApiService();
 
-            Api a = new Api("k", "k", "k");
-
-            bool alreadyExists = service.RegisterHospitalOnPharmacy(a);
+            Api registerFailed = new Api("k", "k", "k");
+            bool alreadyExists = service.RegisterHospitalOnPharmacy(registerFailed);
 
             Assert.False(alreadyExists);
```

Comment:
I would expect that the result of alreadyExists variable is True, so the name is not very fitting, you could just name it result or success